package_manager/dpkg_extract.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+set -o errexit
+
+DEB=$1
+shift
+if [ -z ${DEB+x} ]; then
```

Comment:
From my testing, this doesn't work, as `DEB` is always defined. (That is, even if `$1` is undefined, the script ends up seeing `DEB=`, resulting in defining `DEB` .) I prefer `[ -z "${DEB}" ]` (i.e., just surrounding `DEB` in quotes).

Reply:
Fixed.